examples/userdemo/annotate_explain.py

```diff
@@ -7,28 +7,13 @@
 
 import matplotlib.pyplot as plt
 import matplotlib.patches as mpatches
-from mpl_toolkits.axes_grid1.axes_grid import AxesGrid
-from matplotlib.offsetbox import AnchoredText
 
+
+fig, axs = plt.subplots(2, 2)
```

Comment:
Did you change the shape on purpose?  I don't really mind, but the original was 1x4...

Reply:
Then you'd need to set a figsize (otherwise the subplots are really small), and it just works as well without it after reshaping to (2, 2).  Basically, trying to concentrate the example on the relevant points.

(the connectionstyle example does set the figsize because I don't think it can really be made to work nicely without it).